tests/framework/jailer.py

```diff
@@ -44,10 +43,10 @@ def __init__(
         self,
         jailer_id,
         exec_file,
+        netns,
```

Comment:
Why is that not optional any more? Can't we have jailer environments without network namespace configured?

Reply:
hmm we can by passing `None`, but true. I don't remember why I did this. I will revert.

Reply:
Actually I am not sure how well no network namespace is supported, since no test uses it, but I will leave the support in.

Reply:
I found an instance of `netns=None` added in this PR.